articles/cognitive-services/text-analytics/how-tos/text-analytics-how-to-entity-linking.md

```diff
@@ -180,6 +180,7 @@ All POST requests return a JSON formatted response with the IDs and detected ent
 
 Output is returned immediately. You can stream the results to an application that accepts JSON or save the output to a file on the local system, and then import it into an application that allows you to sort, search, and manipulate the data.
 
+Because of multilingual and emoji support, see also [how to process offsets in API output](text-analytics-how-to-process-offsets)
```

Comment:
Might want to rephrase, and not start the sentence with "because". Something like: "Additional information on how to process offsets for Unicode strings, see ..."